src/content/components/Tooltip.js

```diff
@@ -0,0 +1,129 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import React, { PureComponent } from 'react';
+import ReactDOM from 'react-dom';
+import type { CssPixels } from '../../common/types/units';
+
+require('./Tooltip.css');
+
+const MOUSE_OFFSET = 15;
+
+type Props = {
+  mouseX: CssPixels,
+  mouseY: CssPixels,
+  children?: React$Element<*>,
+}
+
+export default class Tooltip extends PureComponent {
+  props: Props;
+
+  state: {
+    interiorElement: HTMLElement | null,
+    interiorElementRAF: HTMLElement | null,
+  };
+
+  _isMounted: boolean;
+  _mountElement: ?HTMLElement;
+
+  constructor(props: Props) {
+    super(props);
+    (this: any)._getMountElement = this._getMountElement.bind(this);
+    this.state = {
+      interiorElement: null,
+      interiorElementRAF: null,
+    };
+  }
+
+  _getMountElement(el: HTMLElement) {
```

Comment:
I don't really like the name of this method because when I read it I think it will return the mountElement, but it's really the opposite. maybe `_refMountElement` or `_setMountElement`?